src/nwb_datajoint/spikesorting/sortingview.py

```diff
@@ -48,19 +50,33 @@ def make(self, key: dict):
         key['workspace_uri'] = workspace.uri
         key['sortingview_recording_id'] = workspace.add_recording(recording=h5_recording,
                                                                   label=workspace_name)
-        sorting = Curation.get_curated_sorting_extractor(key)
+        sorting = Curation.get_curated_sorting(key)
         sorting = si.create_extractor_from_new_sorting(sorting)
         h5_sorting = sv.LabboxEphysSortingExtractor.store_sorting_link_h5(
             sorting, str(Path(recording_path) / 'sorting.h5'))
-        workspace_uri = (self & key).fetch1('workspace_uri')
-        workspace = sv.load_workspace(workspace_uri)
+
         key['sortingview_sorting_id'] = workspace.add_sorting(recording_id=workspace.recording_ids[0],
-                                                              sorting=h5_sorting)
+                                                              sorting=h5_sorting, label=key['curation_id'])
+        self.insert1(key)
+
         # add metrics to the sorting if they exist
-        metrics = (Curation & key).fetch1('metrics')
+        metrics = (Curation & key).fetch1('quality_metrics')
         self.add_metrics_to_sorting(
             key, metrics, key['sortingview_sorting_id'])
-        self.insert1(key)
+
+        labels = (Curation & key).fetch1('curation_labels')
+        print(labels)
+        for unit_id in labels:
+            for label in labels[unit_id]:
+                workspace.sorting_curation_add_label(
```

Comment:
should we set another flag here to not show noise units (which can be very large)? or is that handled by `sortingview` internally? @magland @jsoules